tests/suite/graceful_recovery_test.go

```diff
@@ -0,0 +1,206 @@
+package suite
+
+import (
+	"fmt"
+	"net/http"
+	"os/exec"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/nginxinc/nginx-gateway-fabric/tests/framework"
+)
+
+var _ = Describe("Graceful Recovery test", Ordered, Label("nfr", "graceful-recovery"), func() {
```

Comment:
Since this test is not enabled in the pipeline, it could help to provide the output of the test run in the description of the PR.

Reply:
If I "move this test to the regular pipeline run" like some of the conversations above, is this comment still relevant?

Reply:
this was just a suggestion to help reviewers.  since the test is not enabled in the pipeline, without fetching it and running it locally, there is no way to see the output that its produced when you `make test ...`